pkg/resource/json_data.go

```diff
@@ -117,7 +117,7 @@ func (r *JSONData) fetchURLs(rootNode jsonNode, clonePath string) error {
 		if err != nil {
 			// Not being able to retrieve a URL found inside JSONData is not a fatal error. Logging until update how
 			// we manage fatal/nonfatal errors flowing through the application.
-			logger.Info("%v", fmt.Errorf("could not fetch url: path=%q url=%q error=%q", leafNode.path, obj, err))
+			r.Warning(logger.CloneError, "%v", fmt.Errorf("could not fetch url: path=%q url=%q error=%q", leafNode.path, obj, err))
```

Comment:
```suggestion
			r.Error(logger.CloneError, "%v", fmt.Errorf("could not fetch url: path=%q url=%q error=%q", leafNode.path, obj, err))
```

Thoughts since were handling an error case, but it's just not critical?